docs/api/specs/desktopAgentCommunicationProtocol.md

```diff
@@ -0,0 +1,448 @@
+
+- [`broadcastRequest`](https://fdc3.finos.org/schemas/next/api/broadcastRequest.schema.json)
+- [`broadcastResponse`](https://fdc3.finos.org/schemas/next/api/broadcastResponse.schema.json)
+
+See [`addContextListener()`](#addcontextlistener) above for the `broadcastEvent` used to deliver the broadcast to other apps.
+
+#### `createPrivateChannel()`
+
+Request and response used to implement the [`createPrivateChannel()`](../ref/DesktopAgent#createprivatechannel) API call:
+
+- [`createPrivateChannelRequest`](https://fdc3.finos.org/schemas/next/api/createPrivateChannelRequest.schema.json)
+- [`createPrivateChannelResponse`](https://fdc3.finos.org/schemas/next/api/createPrivateChannelResponse.schema.json)
+
+#### `findInstances()`
+
+Request and response used to implement the [`findInstances()`](../ref/DesktopAgent#findinstances) API call:
+
+- [`findInstancesRequest`](https://fdc3.finos.org/schemas/next/api/findInstancesRequest.schema.json)
+- [`findInstancesResponse`](https://fdc3.finos.org/schemas/next/api/findInstancesResponse.schema.json)
+
+#### `findIntent()`
+
+Request and response used to implement the [`findIntent()`](../ref/DesktopAgent#findintent) API call:
+
+- [`findIntentRequest`](https://fdc3.finos.org/schemas/next/api/findIntentRequest.schema.json)
+- [`findIntentResponse`](https://fdc3.finos.org/schemas/next/api/findIntentResponse.schema.json)
+
+#### `findIntentsByContext()`
+
+Request and response used to implement the [`findIntentsByContext()`](../ref/DesktopAgent#findintentsbycontext) API call:
+
+- [`findIntentsByContextRequest`](https://fdc3.finos.org/schemas/next/api/findIntentsByContextRequest.schema.json)
+- [`findIntentsByContextResponse`](https://fdc3.finos.org/schemas/next/api/findIntentsByContextResponse.schema.json)
+
+#### `getAppMetadata()`
+
+Request and response used to implement the [`getAppMetadata()`](../ref/DesktopAgent#getappmetadata) API call:
+
+- [`getAppMetadataRequest`](https://fdc3.finos.org/schemas/next/api/getAppMetadataRequest.schema.json)
+- [`getAppMetadataResponse`](https://fdc3.finos.org/schemas/next/api/getAppMetadataResponse.schema.json)
+
+#### `getCurrentChannel()`
+
+Request and response used to implement the [`getCurrentChannel()`](../ref/DesktopAgent#getcurrentchannel) API call:
+
+- [`getCurrentChannelRequest`](https://fdc3.finos.org/schemas/next/api/getCurrentChannelRequest.schema.json)
+- [`getCurrentChannelResponse`](https://fdc3.finos.org/schemas/next/api/getCurrentChannelResponse.schema.json)
+
+#### `getInfo()`
+
+Request and response used to implement the [`getInfo()`](../ref/DesktopAgent#getinfo) API call:
+
+- [`getInfoRequest`](https://fdc3.finos.org/schemas/next/api/getInfoRequest.schema.json)
+- [`getInfoResponse`](https://fdc3.finos.org/schemas/next/api/getInfoResponse.schema.json)
+
+#### `getOrCreateChannel()`
+
+Request and response used to implement the [`getOrCreateChannel()`](../ref/DesktopAgent#getorcreatechannel] API call:
```

Comment:
incorrect link.  We need a closing `)` but we have a closing `]`

Reply:
Fixed - surprised docusaurus didn't complain about that!